clients/frontend-authorization/src/FrontendAuthorizationClient.ts

```diff
@@ -19,7 +19,7 @@ export interface FrontendAuthorizationClient extends AuthorizationClient {
   signOut(requestContext?: ClientRequestContext): Promise<void>;
 
   /** Event called when the user's sign-in state changes - this may be due to calls to signIn(), signOut() or simply because the token expired */
-  readonly onUserStateChanged: BeEvent<(token: AccessToken | undefined) => void>;
+  readonly onUserStateChanged: BeEvent<(token: AccessTokenString | undefined) => void>;
```

Comment:
redundant typing, AccessTokenString = string | undefined

Reply:
I've changed the AccessTokenString type to just be a `string` instead of `sting | undefined`, so this should no longer need a change.